src/traces/heatmap/clean_2d_array.js

```diff
@@ -30,12 +30,23 @@ module.exports = function clean2dArray(zOld, transpose) {
         old2new = function(zOld, i, j) { return zOld[i][j]; };
     }
 
+    var xMap = function(i) {return i;};
```

Comment:
We should use `Lib.identity` here.

Reply:
Done in 59edb5704f076b438e6b8460b3e03097d61748ab